airbyte-integrations/connectors/source-facebook-marketing-api-singer/README.md

```diff
@@ -0,0 +1,66 @@
+# Source Facebook Marketing Api Singer
+
+This is the repository for the Facebook Marketing Api source connector, based on a Singer tap.
+For information about how to use this connector within Airbyte, see [the User Documentation](https://docs.airbyte.io/integrations/sources/facebook-marketing-api).
+
+## Local development
+### Build
+First, build the module by running the following from the `airbyte` project root directory: 
+```
+./gradlew :airbyte-integrations:connectors:source-facebook-marketing-api-singer:build
+```
+
+This will generate a virtualenv for this module in `source-facebook-marketing-api-singer/.venv`. Make sure this venv is active in your
+development environment of choice. If you are on the terminal, run the following
+```
+cd airbyte-integrations/connectors/source-facebook-marketing-api # cd into the connector directory
+source .venv/bin/activate
+```
+If you are in an IDE, follow your IDE's instructions to activate the virtualenv. 
+
+**All the instructions below assume you have correctly activated the virtualenv.**.
+
+### Locally running the connector
+```
+python main_dev.py spec
+python main_dev.py check --config sample_files/test_config.json
+python main_dev.py discover --config sample_files/test_config.json
+python main_dev.py read --config sample_files/test_config.json --catalog sample_files/test_catalog.json
+```
+
+### Unit Tests
+To run unit tests locally, from the connector root run:
+```
+pytest unit_tests
```

Comment:
worth mentioning the gradle command as well?

Reply:
This section is meant more for the "vanilla" python workflow so it doesn't mention gradle, although your comment is making me realize we really should have a section describing how this submodule works as part of the larger monorepo.